SonarQube.TeamBuild.PreProcessor/AnalysisConfigGenerator.cs

```diff
@@ -41,6 +46,10 @@ public static class AnalysisConfigGenerator
             {
                 throw new ArgumentNullException("analyzerSettings");
             }
+            if (programmaticProperties == null)
+            {
+                throw new ArgumentNullException(nameof(programmaticProperties));
```

Comment:
Oops - I updated the snippet I use to add argument checks to use _nameof_. However, we're still building using 2013 in AppVeyor so this causes a build break. I'll change it to use a string instead.